### DISCUSSION TITLE
Exact 90, 180, 270 degree rotation -- no interpolation

### DISCUSSION THREAD

**Discussion by fmw42** [ImageMagick/ImageMagick#5923] [Development] [answered]
### ImageMagick version

7.1.0.57

### Operating system, version and so on

Mac OSX Monterey

### Description

Implement non-interpolating, exact 90, 180, 270 rotations:  -rotate90, -rotate180, -rotate 270

**Comment by snibgo** [comment, marked as answer]
As far as I can see, "`-rotate X`", where X is -90, 0, +90, +180, +270 etc will simply move pixels, and will not interpolate pixels.

If you think otherwise, please show an example.

**Reply by fmw42** [reply]
No, I just assumed that it would interpolate.  Thanks for checking.

This is closed
